RecoLocalTracker/SiPixelRecHits/python/PixelCPEGeneric_cfi.py

```diff
@@ -59,6 +59,11 @@
     MagneticFieldRecord = cms.ESInputTag(""),
 )
 
+# This customizes the Run3 Pixel CPE generic reconstruction in order to activate the IrradiationBiasCorrection
+# because of the expected resolution loss due to radiation damage
+from Configuration.Eras.Modifier_run3_common_cff import run3_common
+run3_common.toModify(PixelCPEGenericESProducer, IrradiationBiasCorrection = cms.bool(True))
```

Comment:
please drop type specifications for all parameters which already exist.
This is a safer way to protect from parameter name mistakes and will also help in possible parameter migrations.

Reply:
done